index.js

```diff
@@ -27,31 +54,41 @@ const postData = (post) => {
 		data: post
 	})
 		.then(function(response) {
+			getData()
 			console.log(response);
 		})
 		.catch(function(error) {
 			console.log(error);
 		});
 };
 
-const getData = () => {
-	axios
-		.get('http://127.0.0.1:5000/')
-		.then((response) => {
-			console.log('I have picked the data');
-			dbData = response.data;
+const deleteData=(post)=>{
+	axios({
+		method: 'delete',
+		url: `http://127.0.0.1:5000/delatePost/${post.id}`,
+		data: post
+	})

```

Comment:
```suggestion
	axios.delete(`http://127.0.0.1:5000/deletePost/${post.id}`, post)
```
Also to note the spelling of `delete ` is wrong in your initial implementation. 

Also it would be wise to create an axios instance from with the baseURL already set.